src/containerapp/azext_containerapp/_help.py

```diff
@@ -162,6 +162,14 @@
 
 """
 
+helps['containerapp revision copy'] = """
```

Comment:
Is this because we missed this In the existing help?

Reply:
Not related  to this change but Line 156-157 of help " az containerapp revision copy -n MyContainerapp -g MyResourceGroup \\
              --cpu 0.75 --memory 1.5Gi" Does the --memory need the value with Units? what happens if one gives just the value? we should clarify this in examples.

Reply:
There are error messages to handle this, I'll add additional helptext to clarify 
<img width="959" alt="image" src="https://user-images.githubusercontent.com/14339314/162791716-70c58cb5-b5bd-4fd8-91b5-c44b740aab35.png">

Reply:
Surface the API error message - we can do this later.

Reply:
fixed

Reply:
Made the `--memory` helptext more clear